src/logs.rs

```diff
@@ -245,14 +242,13 @@ impl TestnetDeploy {
             return Err(Error::EnvironmentDoesNotExist(name.to_string()));
         }
```

Comment:
I cannot highlight all the lines that I want here, so I'll paste them here.

```rust
        let genesis_inventory_path =
            PathBuf::from("inventory").join(format!(".{name}_genesis_inventory_digital_ocean.yml"));
        let remaining_nodes_inventory_path =
            PathBuf::from("inventory").join(format!(".{name}_node_inventory_digital_ocean.yml"));
        if !genesis_inventory_path.exists() || !remaining_nodes_inventory_path.exists() {
            return Err(Error::EnvironmentDoesNotExist(name.to_string()));
        }
```
Can we use `AnsibleRunner::get_inventory_path` to validate these paths, now that we have moved everything to concrete enums?

Reply:
Also there are a couple such validations in the codebase and we can probably clean them up as well?

Reply:
Yeah, it would be possible to centralise those checks. Thanks.